### ISSUE TITLE
FEATUERE REQUEST Clearing all web site data

### ISSUE THREAD

**Issue by mitvitaminen** [mrdokenny/Cookie-AutoDelete#150]
Hi,
it would be nice if you could clear the website settings + cache for the site of the deleted cookie as well. Thank you
Chris

**Comment by ke-d** [comment]
Duplicate of #44 , but mostly waiting on APIs that allow me to delete website data by `hostname`
